x-pack/test/security_solution_endpoint/apps/endpoint/endpoint_list.ts

```diff
@@ -33,27 +33,17 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       'Actions',
     ],
     [
-      'Host-dpu1a2r2yi',
+      'Host-9qenwrl9ko',
       'x',
       'x',
       'Warning',
-      'macOS',
-      '10.2.17.24, 10.56.215.200,10.254.196.130',
-      'x',
-      'x',
-      '',
-    ],
-    [
-      'Host-rs9wp4o6l9',
-      'x',
-      'x',
-      'Success',
       'Linux',
-      '10.138.79.131, 10.170.160.154',
+      '10.56.228.101, 10.201.120.140,10.236.180.146',
```

Comment:
Can you briefly explain what is the purpose of this? Is this autogenerated?

Reply:
These are autogenerated and change every time we update document generator. This is not part of this PR though ;)